src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -368,18 +371,39 @@ export default class SpecialResolutionForm extends Mixins(DateMixin) {
       ...this.getcreateResolution,
       signingDate: val
     })
+    this.validate()
   }
   /** called to store signing party. */
-   @Watch('signingPerson', { deep: true })
-  protected async onSigningPersonChanged (): Promise<void> {
+   @Watch('signatory', { deep: true })
+  protected async onsignatoryChanged (): Promise<void> {
     this.setResolution({
       ...this.getcreateResolution,
-      signingPerson: this.signingPerson
+      signatory: this.signatory
     })
+    await this.$nextTick()
+    this.validate()
   }
-  /** Set values if exist */
+   /** called to store component validity to store. */
+   protected async validate () {
+     // wait to reflect validation state
+     await this.$nextTick()
+     // date validation
+     const isResolutionDateValid = this.$refs?.resolutionDatePickerRef?.isDateValid()
+     const isSignatureDateValid = this.$refs?.signatureDatePickerRef?.isDateValid()
```

Comment:
I pulled the PR and found out there is an error in `isDatevalid()`. If it's not in your end, then it's fine.